spec/filewatch/rotate_spec.rb

```diff
@@ -73,7 +73,8 @@ module FileWatch
             FileUtils.mv(directory.join("1.logtmp").to_path, file1_path)
           end
           .then("wait for expectation") do
-            wait(2).for{listener1.calls}.to eq([:open, :accept, :accept, :accept])
+            sleep(0.25) # if ENV['CI']
```

Comment:
I don't understand the meaning of the comment

Reply:
`sleep` is only necessary on "slow" machines such as CI.
seems a delete operation might still be pending while we reach here.
kept a comment as a reminder ...

Reply:
@kares we could actually detect this with `if ENV['CI']` according to [travis documentation](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables)

Reply:
@jsvd yy but I still prefer not to have CI conditions around unless "really" necessary.
left a comment there as a "hint" for the next person wondering why the `sleep` :wilted_flower: